[2024-02-01T22:24:31.736+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:24:29.832465+00:00 [queued]>
[2024-02-01T22:24:31.742+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:24:29.832465+00:00 [queued]>
[2024-02-01T22:24:31.742+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-01T22:24:31.752+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): tf_to_df> on 2024-02-01 22:24:29.832465+00:00
[2024-02-01T22:24:31.757+0000] {standard_task_runner.py:60} INFO - Started process 103 to run task
[2024-02-01T22:24:31.758+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'challenger_leaderboard', 'tf_to_df', 'manual__2024-02-01T22:24:29.832465+00:00', '--job-id', '277', '--raw', '--subdir', 'DAGS_FOLDER/python_tft_tier.py', '--cfg-path', '/tmp/tmpftsmzrzo']
[2024-02-01T22:24:31.759+0000] {standard_task_runner.py:88} INFO - Job 277: Subtask tf_to_df
[2024-02-01T22:24:31.826+0000] {task_command.py:423} INFO - Running <TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:24:29.832465+00:00 [running]> on host 964619f82b50
[2024-02-01T22:24:31.935+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='challenger_leaderboard' AIRFLOW_CTX_TASK_ID='tf_to_df' AIRFLOW_CTX_EXECUTION_DATE='2024-02-01T22:24:29.832465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-01T22:24:29.832465+00:00'
[2024-02-01T22:24:32.018+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-01T22:24:32.035+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=challenger_leaderboard, task_id=tf_to_df, execution_date=20240201T222429, start_date=20240201T222431, end_date=20240201T222432
[2024-02-01T22:24:32.091+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-01T22:24:32.113+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
